@xen-orchestra/web/src/components/host/dashboard/alarms/HostDashboardAlarms.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <UiCard class="host-dashboard-alarms">
+    <UiCardTitle>
+      {{ t('alarms') }}
+      <UiCounter
+        v-if="hostAlarms.length !== 0 && areHostAlarmsReady"
+        accent="danger"
+        size="small"
+        variant="primary"
+        :value="hostAlarms.length"
+      />
+    </UiCardTitle>
+    <VtsLoadingHero v-if="!areHostAlarmsReady" type="card" />
+    <VtsAllGoodHero v-else-if="hostAlarms.length === 0" type="card" />
+    <div v-else class="alarm-list-container">
+      <UiAlarmList>
+        <template v-for="alarm in hostAlarms" :key="alarm.id">
+          <UiAlarmItem
+            :label="alarm.body.name"
+            :percent="Number(alarm.body.value)"
+            :size="uiStore.isDesktopLarge ? 'large' : 'small'"
+            :date="alarm.time"
+          >
+            <template #link>
+              <AlarmLink :type="alarm.object.type" :uuid="alarm.object.uuid" />
+            </template>
+          </UiAlarmItem>
+        </template>
```

Comment:
Here I think `<template>` is unnecessary. What do you think?

```suggestion
        <UiAlarmItem
           v-for="alarm in hostAlarms" :key="alarm.id"
          :label="alarm.body.name"
          :percent="Number(alarm.body.value)"
          :size="uiStore.isDesktopLarge ? 'large' : 'small'"
          :date="alarm.time"
        >
          <template #link>
            <AlarmLink :type="alarm.object.type" :uuid="alarm.object.uuid" />
          </template>
        </UiAlarmItem>
```